src/models/meer7/repairs.md

```diff
@@ -0,0 +1,336 @@
+3. Unscrew the two screws holding the expansion module in from the back of the system.
+
+![I/O expansion module (external view)](./img/expansion-module-external.webp)
+
+4. Lift the expansion module out of the system.
+    - When reinstalling the expansion module, the side labeled `BTM` should face outwards (so it will point down towards the bottom of the machine once the system is upright).
+
+### Steps to remove the expansion slot placeholder:
+
+1. Unscrew the two screws holding the expansion slot placeholder in from the back of the system.
+
+![Expansion slot placeholder](./img/expansion-placeholder.webp)
+
+2. Lift the expansion slot placeholder out of the system.
+    - When reinstalling the slot placeholder, the side labeled `BTM` should face outwards (so it will point down towards the bottom of the machine once the system is upright).
+
+## Removing the motherboard:
+
+Removing the motherboard is necessary for replacing the CMOS battery, cooling fan, and thermal paste.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to remove the motherboard:
+
+1. Follow the steps above to [remove the bottom panel](#removing-the-bottom-panel) and [remove the M.2 drive](#replacing-an-m2-drive).
+2. If your Meerkat is the tall variety:
+    - Remove the [optional I/O add-on or expansion slot placeholder](#removing-the-optional-io-add-on).
+    - Disconnect the ribbon cable that leads to the 2.5" drive bay.
+        - The white latch pulls away from the motherboard to free the cable; gently pull the cable out of the connector.
+
+![SATA drive bay ribbon cable connector](./img/sata-ribbon-connector.webp)
+
+3. Disconnect the two [wireless antenna cables](#replacing-the-wireless-card).
+4. Remove the two motherboard mounting screws along the back edge of the machine.
+
+![Motherboard mounting screws](./img/motherboard-screws.webp)
+
+5. Carefully lift the motherboard out of the case.
+    - It is easiest to lift the front side first, pivoting the board against the back side of the case.
+    - [Removing the RAM](#replacing-the-ram) is not required, but can make removing the motherboard easier.
+    - Slightly squeezing the front sides of the chassis can help provide more clearance while pivoting the motherboard.
+
+## Replacing the fan:
+
+The cooling fan can be removed to clean dust out of the cooling system or to access the CPU heatsink when replacing the thermal paste.
+
+The screws used for the fan are the same type as those used for the [M.2 SSDs](#replacing-an-m2-drive).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 25 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fan:
+
+1. Follow the steps above to [remove the motherboard](#removing-the-motherboard).
```

Comment:
The motherboard section starts with removing the bottom panel, NVMe drive, and optional IO expansion, but the rest of the sections list out every component that needs to be removed in the first step. I'm okay with this since the motherboard section has all the needed info, but wanted to point it out in case you'd rather make it more consistent with the rest.